contracts/src/token/psp22/utils/token_timelock.rs

```diff
@@ -141,7 +151,11 @@ impl<T: Storage<Data>> Internal for T {
         Ok(())
     }
 
-    default fn _token(&mut self) -> &mut PSP22Ref {
+    fn _token(&mut self) -> &mut PSP22Ref {
         &mut self.data().token
     }
+
+    fn _beneficiary(&self) -> AccountId {
```

Comment:
why this method was added?

Reply:
the method is called in internal fn. Forgot to call it in the external)